File: TS_JT-Reader/TS_JT-Reader_Vis-Studio/TS_JT-Reader/DeeringCode.ts
module C_sharp_JT_Reader.Codecs {
    export class DeeringCode {
        m_sextant: number = 0;
        m_octant: number = 0;
        m_theta: number = 0;
        m_psi: number = 0;
        constructor(sextant: number, octant: number, theta: number, psi: number) {
            this.m_sextant = sextant;
            this.m_octant = octant;
            this.m_theta = theta;
            this.m_psi = psi;
        }
    }
}

File: TS_JT-Reader/C#_JT-Reader_Vis-Studio/C sharp JT-Reader/Codecs/DeeringCode.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace C_sharp_JT_Reader.Codecs
{
    public class DeeringCode
    {
        long m_sextant = 0;
        long m_octant = 0;
        long m_theta = 0;
        long m_psi = 0;

        public DeeringCode(long sextant, long octant, long theta, long psi)
        {
            m_sextant = sextant;
            m_octant = octant;
            m_theta = theta;
            m_psi = psi;
        }
    }
}